buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -65,8 +65,10 @@ class ClusterFormationTasks {
         project.configurations {
             elasticsearchDistro
         }
+        // TODO why? why? why? why? why? why? why? why? why? why? why? why? why? why? why?
+        String artifactId = ['rpm', 'deb'].contains(distro) ? distro : 'elasticsearch'
```

Comment:
I'm sure that's super obvious to someone who set this up but its really quite obfuscated to me. I'll fix it.